linux_os/guide/system/accounts/accounts-session/accounts_tmout/oval/shared.xml

```diff
@@ -1,10 +1,16 @@
 <def-group>
```

Comment:
I think these `accounts_tmout` can be removed as per: https://github.com/ComplianceAsCode/content/pull/6631#discussion_r584669502

Reply:
Let's leave then. As discussed in another topic.